pkg/core/ioutil/dump/dump.go

```diff
@@ -21,7 +21,7 @@ const (
 	JSONFormat  = "json"
 	YAMLFormat  = "yaml"
 	YMLFormat   = "yml"
-	EmptyFormat = ""
+	EmptyFormat = "none"
```

Comment:
This change is causing issues. It is bypassing the tabular view in multiple commands.

> ./rhoas service-registry role list => returns nil
> ./rhoas service-registry role list -o none => returns table

Reply:
See my comment above. It is basically bug. @gowriswarupk please add debugging statement for format to see why this is happening - we can remove them later.